system-tests/__snapshots__/new_project_spec.js

```diff
@@ -39,7 +40,8 @@ exports['e2e new project passes 1'] = `
   (Video)
 
   -  Started processing:  Compressing to 32 CRF                                                     
-  -  Finished processing: /XXX/XXX/XXX/cypress/videos/app_spec.js.mp4                     (X second)
+  -  Finished processing: /XXX/XXX/XXX/cypress/videos/cypress/integration/app_spec.js     (X second)
```

Comment:
I know this probably correct, but it somehow feels off... `cypress/videos/cypress/integration` vs `cypress/videos/integration`

Reply:
`cypress/videos` + `<spec relative to projectRoot>`

Previously we stripped of `integrationFolder`. Yes this feels weird, I agree.

https://cypressio.slack.com/archives/CNM0FKJFJ/p1638886882133300?thread_ts=1638886612.133100&cid=CNM0FKJFJ